common/djangoapps/student/views.py

```diff
@@ -2552,13 +2556,18 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
 
     if request.method == 'POST':
         password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
-        if not is_password_valid:
+        is_security_violated, password_err_msg = validate_password(user, password)
```

Comment:
@schenedx The reason why I did this is, we have two different scenarios

1. length is not valid
2. security policy is violated (e.g resetting too frequently)

For both of the above problems have different error page.

1. show error message and show form so that user can re-enter password with valid length.
2. show error message and don't show the form again, because user is not allowed to reset his password.